front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -198,17 +182,6 @@ const AdminProjectsProjectGeneral = ({
     return file !== null;
   }
 
-  useEffect(() => {
-    if (!isNilOrError(areas)) {
-      setAreasOptions(
-        areas.map((area) => ({
-          value: area.id,
-          label: localize(area.attributes.title_multiloc),
-        }))
-      );
-    }
-  }, [areas, localize]);
```

Comment:
The localize and publicationStatus (higher up) in the useEffect were causing the frozen pages. I realized we don't need areasOptions in the state to begin with. I also thought it's better to have all that inside the GeographicAreas component. It also allowed me to clean up some pretty confusing code (the areaIds and areaValues below) and keep this component more focused. I think it's the direction we should take for this entire form.